src/dispatchers.ts

```diff
@@ -81,7 +81,6 @@ export const keypairDispatcher = (
         ctx: Context<ContextData>,
         handle: string,
     ) {
-        if (handle !== ACTOR_DEFAULT_HANDLE) return [];
```

Comment:
Should we do this for the Actor dispatcher too?

Reply:
Sure, updated in https://github.com/TryGhost/ActivityPub/pull/535/commits/dc5de38ef4bc2dd57cf3b51f970dd6b4a2101ed8